tasks/kernel_matrix_testing/env-setup.sh

```diff
@@ -20,15 +20,16 @@ apt install -y \
     rpcbind
 
 if [ "$(uname -m )" == "aarch64" ]; then
-    apt install -y qemu-efi-aarch64
+    sudo apt install -y qemu-efi-aarch64
 fi
 
-systemctl start nfs-kernel-server.service
+sudo systemctl start nfs-kernel-server.service
 
 pip install -r tasks/kernel_matrix_testing/requirements.txt
```

Comment:
May want to make this explicitly `pip3` since some machines may have both python 2 and 3.